src/bg/background.js

```diff
@@ -188,6 +192,20 @@ function badgeMetric(metric, value, tabid) {
           tabId: currentTab,
         });
         break;
+      case 'inp':
+          chrome.browserAction.setIcon({
+            path: '../../icons/slow128w-fid.png', // TODO: make an icon for INP
```

Comment:
@addyosmani do you have a source file for editing these icons?

Or would it make more sense to exclude INP from the icons given that it's not a CWV? For example, we don't show FCP failures in the icon.

Reply:
@mmocny I'd be inclined to roll back these icon changes for INP for consistency with the FCP behavior. WDYT?

Reply:
Okay. So, it's actually a bit more complicated than I expected.

There is both a background color, and a metric-specific badging.  When any of the metrics fail to meet threshold, the extension changes BG color, then it will loop through the metrics to badge each one.

I can remove the badging, but the extension still goes red when INP doesn't meet threshold -- I assume we want to remove all the BIG RED warnings for INP to make it obviously optional and not appear to fail vitals?  I'll see if I can do that.

Reply:
Done.